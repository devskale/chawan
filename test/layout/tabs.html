<!doctype html>
<pre style="background-color: red">
x	y
x    	y
x       y
^ The above rows should be aligned, and dump mode should output spaces.
</pre>
Without bgcolor:
<pre>
x	y
x    	y
x       y
^ The above rows should be aligned, and dump mode should output real tab
characters.
</pre>
