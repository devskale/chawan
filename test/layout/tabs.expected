
[48;2;255;0;0mx       y                                                                       [49m
[48;2;255;0;0mx       y                                                                       [49m
[48;2;255;0;0mx       y                                                                       [49m
[48;2;255;0;0m^ The above rows should be aligned, and dump mode should output spaces.         [49m

Without bgcolor:

x	y
x    	y
x       y
^ The above rows should be aligned, and dump mode should output real tab
characters.
